lib/gds_api/locations_api.rb

```diff
@@ -12,7 +12,8 @@ def local_custodian_code_for_postcode(postcode)
 
     return [] if response["results"].nil?
 
-    response["results"].map { |r| r["local_custodian_code"] }.uniq
+    # The local custodian code 7655 is Ordnance Survey added data, so it should be ignored
+    response["results"].map { |r| r["local_custodian_code"] }.reject { |code| code == 7655 }.uniq
```

Comment:
Ooh, this looks interesting. I don't really understand the reason behind this change - could you add more detail to the commit?

E.g.

- Where is this Ordnance Survey data being added?
  - For which postcode(s)?
  - Why is it being added? What does it mean?
  - Why do we need to ignore it?
- How did you discover it? Are there potentially other codes we'll need to filter out? (Might affect the implementation)

Reply:
Actually I was just looking over a pdf from ordnance survey. There is also the code 11 for HIGHWAYS ENGLAND. Now I'm not sure if this should be removed here, or in locations-api where we store the OSApi response 🤔

Reply:
Agreed, sounds like something best handled in Locations API itself.